deploy/cognito/userpool.tf

```
@@ -0,0 +1,38 @@
+variable env {}
+
+resource "aws_cognito_user_pool" "users" {
+  name = "hallebarde-${var.env}"
+
+  admin_create_user_config {
+    allow_admin_create_user_only = true
+  }
+
+  mfa_configuration = "OFF"
+
+  schema {
+    attribute_data_type = "String"
+    name = "email"
+    required = true
+    mutable = false
+
+    string_attribute_constraints {
+      max_length = 100
+      min_length = 1
+    }
+  }
+
+  schema {
+    attribute_data_type = "String"
+    name = "phone"
+    required = false
+    mutable = false
+
+    string_attribute_constraints {
+      max_length = 15
+      min_length = 3
```

Also, I would feel it more natural to read with the minimum before the maximum, like
```suggestion
      min_length = 3
      max_length = 15
```

---

Same comment for lines 19 & 20, what do you think ?

---

OK